### ISSUE TITLE
[Buttons] Top half of FAB circle is not visible when Smart Invert Colors enabled

### ISSUE THREAD

**Issue by ianegordon** [material-components/material-components-ios#3163]
b/74075697

**Comment by romoore** [comment]
Doesn't this depend on the colors chosen for the fab and background?

**Comment by romoore** [comment]
Internal-only issue.
